### ISSUE TITLE
Remove link to input.mozilla.org

### ISSUE THREAD

**Issue by flekkowich** [mozilla/kitsune#4498]
Current: https://support.mozilla.org/en-US/kb/rate-your-firefox-experience-heartbeat

Future: Remove all links to input.mozilla.org, including --> Share feedback about Firefox through Input. Please mention heartbeat in your comment.

**Comment by rtanglao** [comment]
@akatsoulas @LeoMcA there are no links to input.mozilla.org in the Kitsune code base right? just in the content of SUMO KB articles I would guess!

**Comment by akatsoulas** [comment]
@rtanglao that's correct. I did a search and I couldn't find anything related in the code. Just the KB article. Tagging also @jonichan

**Comment by AliceWyman** [comment]
Joni removed the Input link from the https://support.mozilla.org/en-US/kb/rate-your-firefox-experience-heartbeat article - see https://support.mozilla.org/en-US/kb/rate-your-firefox-experience-heartbeat/history

@jonichan There's also a link to https://qsurvey.mozilla.com/s3/FirefoxInput/ in this common forum response:
https://support.mozilla.org/en-US/kb/forum-response-volunteers

**Comment by AliceWyman** [comment]
@jonichan The Input link in https://support.mozilla.org/en-US/kb/forum-response-volunteers was removed by KiKi on Dec 4, 2020, However, there's also a https://input.mozilla.org/feedback/ link in another contributor article:
https://support.mozilla.org/en-US/kb/answering-questions-support-forum#w_off-topic-questions

**Comment by AliceWyman** [comment]
@jonichan I removed (commented out) the "Help us make Firefox better" section in the https://support.mozilla.org/en-US/kb/install-older-version-firefox "How to" article, since it included a link to https://qsurvey.mozilla.com/s3/FirefoxInput/
